README.md

```diff
@@ -4,28 +4,20 @@ Mattermost is an open source, self-hosted Slack-alternative https://mattermost.o
 
 It's written in Golang and React and runs as a single Linux binary with MySQL or Postgres. Every month on the 16th [a new compiled version is released under an MIT license](https://about.mattermost.com/download/).
 
-This project hosts plugins built by Mattermost staff and community. They are categorized as follows:
+[index.md](index.md) is the list of plugins built by Mattermost staff and community. They are categorized as follows:
 
  - **Production**: Certified by the Mattermost team, and tested prior to each Mattermost server release.
  - **Unofficial**: Non-production plugins built by Mattermost staff and community.
 
 ### Add your plugin
 
-To add your plugin as an unofficial plugin:
+To add your plugin as an unofficial plugin submit a pull request with a reference of your plugin included in [index.md](index.md)
 
 1. Fork this repository and create a branch in your fork.
-2. On your branch run:
-  ```bash
-  git submodule add <repo-link> Unofficial/<repo-name>
-  ```
-  For example,
-  ```bash
-  git submodule add https://github.com/mattermost/mattermost-plugin-profanity-filter.git Unofficial/mattermost-plugin-profanity-filter
-  ```
+2. On your branch, modeify index.md.
```

Comment:
Given it's just a PR that edits index.md, wondering if steps 1-4 can be removed altogether.

Reply:
Agree, this feels unnecessarily verbose. (If they can write a plugin, I think they can handle the PR to include theirs here.)